parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -117,6 +117,7 @@ $global:PrimaryAvailabilitySetName = "{{GetVariable "primaryAvailabilitySetName"
 $global:PrimaryScaleSetName = "{{GetVariable "primaryScaleSetName"}}"
 
 $global:KubeClusterCIDR = "{{GetParameter "kubeClusterCidr"}}"
+$global:KubeClusterCIDRs = "{{GetParameter "kubeClusterCidrs"}}"

```

Comment:
Could we reuse `KubeClusterCIDR` since it supports multiple cidrs with `,` by design?
$exceptionAddresses += $subnetToPass[0] was added in https://github.com/Azure/AgentBaker/commit/fa17dfd271a7e8c060be70d5c214b35ca067c33c. So you only need to update `$exceptionAddresses += $subnetToPass[0]` to 
```
foreach ($cidr in $subnetToPass) {
                $exceptionAddresses += $cidr
            }
```

Reply:
With this change, it will block the current users even without new cse script package.

Reply:
yeah, latest update just re-uses kubeClusterCidr